scout/server/blueprints/variant/utils.py

```diff
@@ -41,13 +42,20 @@ def add_panel_specific_gene_info(panel_info):
         if gene_info.get("mosaicism"):
             mosaicism = True
 
+        if gene_info.get("comment"):
```

Comment:
It would be perhaps easier to handle if comment were a list, but this works

Reply:
Sure, good point; I was trying to stick to the db model, but no need really, it's already a little different for the other keys.